src/main/query_profiler.cpp

```diff
@@ -454,6 +442,71 @@ string QueryProfiler::QueryTreeToString() const {
 	return str.str();
 }
 
+string RemovePhaseTimingName(string name, string metric) {
+    return metric.substr(name.size());
+}
```

Comment:
Why do we need this?

Reply:
For the `profiling_tree` output, it's too long otherwise. So it removes the `optimizer` or `planner` etc. 

```
┌─────────────────────────────────────┐
│┌───────────────────────────────────┐│
││    Query Profiling Information    ││
│└───────────────────────────────────┘│
└─────────────────────────────────────┘
select unnest(['Maia', 'Thijs', 'Mark', 'Hannes', 'Tom', 'Max', 'Carlo', 'Sam', 'Tania']) as names order by random();
┌────────────────────────────────────────────────┐
│┌──────────────────────────────────────────────┐│
││              Total Time: 0.0351s             ││
│└──────────────────────────────────────────────┘│
└────────────────────────────────────────────────┘
┌────────────────────────────────────────────────┐
│            Optimizer timing: 0.0033s           │
│┌──────────────────────────────────────────────┐│
││     Build Side Probe Side Timing: 0.0000s    ││
││        Column Lifetime Timing: 0.0001s       ││
││       Common Aggregate Timing: 0.0000s       ││
││     Common Subexpressions Timing: 0.0000s    ││
││  Compressed Materialization Timing: 0.0000s  ││
││       Cte Filter Pusher Timing: 0.0000s      ││
││          Deliminator Timing: 0.0000s         ││
││       Duplicate Groups Timing: 0.0000s       ││
││      Expression Rewriter Timing: 0.0012s     ││
││           Extension Timing: 0.0000s          ││
││         Filter Pullup Timing: 0.0000s        ││
││        Filter Pushdown Timing: 0.0001s       ││
││           In Clause Timing: 0.0000s          ││
││     Join Filter Pushdown Timing: 0.0000s     ││
││          Join Order Timing: 0.0003s          ││
││        Limit Pushdown Timing: 0.0000s        ││
││       Materialized Cte Timing: 0.0000s       ││
││          Regex Range Timing: 0.0000s         ││
││        Reorder Filter Timing: 0.0000s        ││
││    Statistics Propagation Timing: 0.0002s    ││
││             Top N Timing: 0.0000s            ││
││        Unnest Rewriter Timing: 0.0000s       ││
││        Unused Columns Timing: 0.0001s        ││
│└──────────────────────────────────────────────┘│
└────────────────────────────────────────────────┘
┌────────────────────────────────────────────────┐
│        Physical planner timing: 0.0002s        │
│┌──────────────────────────────────────────────┐│
││        Column Binding Timing: 0.0001s        ││
││          Create Plan Timing: 0.0001s         ││
││         Resolve Types Timing: 0.0001s        ││
│└──────────────────────────────────────────────┘│
└────────────────────────────────────────────────┘
┌────────────────────────────────────────────────┐
│             Planner timing: 0.0028s            │
│┌──────────────────────────────────────────────┐│
││            Binding Timing: 0.0014s           ││
│└──────────────────────────────────────────────┘│
└────────────────────────────────────────────────┘
```

Reply:
Although it will also be shorter with `_timing` removed.

Reply:
Oh actually I forgot to remove it, because I'm simplified the code after I wrote this. Oops!